lib/AvatarGroup/index.js

```diff
@@ -17,28 +17,42 @@ class AvatarGroup extends Component {
   }
 
   render() {
-    const { children, maximum } = this.props;
-    const total = children.length;
-    const display = children.slice(0, maximum);
-    const remaining = children.slice(maximum, total);
+    const { children, maximum, className, colours } = this.props;
+    const total = children && children.length || 0;
```

Comment:
If there was an issue with rendering a single component I would recommend adding this into the test file (rendering multiple & a single component).